### PR TITLE
modifying new timeout values for power architecture

### PR DESCRIPTION
**Pull request by AaruniAggarwal**
Signed-off-by: Aaruni Aggarwal <aaruniagg@gmail.com>

### CODE REVIEW

File: ocs_ci/ocs/ocp.py [hunk 1]
```diff
@@ -681,6 +681,10 @@ def wait_for_delete(self, resource_name="", timeout=60, sleep=3):
             bool: True in case resource deletion is successful
 
         """
+        if config.ENV_DATA["platform"].lower() == constants.IBM_POWER_PLATFORM:
+            timeout = 720
+        else:
+            timeout = 60
```
Comment (vavuthu) [inline ocs_ci/ocs/ocp.py:687]: this overwrite the timeout if user want to pass timeout as parameter

File: ocs_ci/utility/utils.py [hunk 2]
```diff
@@ -1577,6 +1577,10 @@ def ceph_health_check(namespace=None, tries=20, delay=30):
             delay of 30 seconds if default values are not changed via args.
 
     """
+    if config.ENV_DATA["platform"].lower() == constants.IBM_POWER_PLATFORM:
+        delay = 60
+    else:
+        delay = 30
```
Comment (vavuthu) [inline ocs_ci/utility/utils.py:1583]: this overwrite the delay if user want to pass delay as parameter

File: ocs_ci/ocs/ocp.py (hunk 1 above)
Comment (AaruniAggarwal) [reply ocs_ci/ocs/ocp.py]: I will remove the else condition so that it won't affect other platforms

Comment (AaruniAggarwal) [reply ocs_ci/ocs/ocp.py]: For IBM Power platform resources are not getting deleted in 60 sec . We are getting many timeout failures due to resources like PVCs not getting deleted in mentioned timeout value.

Comment (AaruniAggarwal) [reply ocs_ci/ocs/ocp.py]: @vasukulkarni ^^

Comment (AaruniAggarwal) [reply ocs_ci/ocs/ocp.py]: @vavuthu ^^

**Review by vavuthu — APPROVED** [review]

**Comment by AaruniAggarwal** [comment]
@vasukulkarni 
Could you please review this PR .
Thanks in advance.

**Review by vasukulkarni — APPROVED** [review]

**Comment by vasukulkarni** [comment]
Please added Fixes: in the description so that its clear why we are increasing the timeout and what tests are failing

**Pull request merged**